api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -611,6 +611,40 @@ type FelixConfigurationSpec struct {
 	// [Default: Auto]
 	BPFConntrackCleanupMode *BPFConntrackMode `json:"bpfConntrackMode,omitempty" validate:"omitempty,oneof=Auto Userspace BPFProgram"`
 
+	// BPFConntrackTimers overides the default values for the specified conntrack timer if
+	// set. It is a key-value make, where each value can be either a duration or `auto` to
+	// pick the value from a Linux conntrack timeout.
+	//
+	// Possible values for the keys are: CreationGracePeriod, TCPPreEstablished,
+	// TCPEstablished, TCPFinsSeen, TCPResetSeen, UDPLastSeen, GenericIPLastSeen,
+	// ICMPLastSeen.
+	//
+	// Unset or incorrect values are replaced by the default values with a warning log for
+	// incorrect values.
+	//
+	// Current auto mappings:
+	//
+	// TCPPreEstablished: nf_conntrack_tcp_timeout_syn_sent
```

Comment:
Would it make sense to call this `TCPSynSent` to align with the underlying timer?

Reply:
And same in general. I'm not an expert on these, so take with a grain of salt, but seems more intuitive if we don't need to refer to this mapping to know what timers are being set.

Reply:
I would make sense to rename the TCP timer, but the linux timers are not quite 1:1 to our timers so I thought it would be better if we spell out which timer do match. I originally allowed users to pick their own mapping but we decided that it is not the best and we hardcoded the mappings.